crates/ghost_actor/tests/manual_example_mod/test_protocol.rs

```diff
@@ -170,7 +172,7 @@ pub trait TestActorRef<'lt, X: 'lt + Send + Sync>: GhostEndpoint<'lt, X, TestPro
 }
 
 impl<'lt, X: 'lt + Send + Sync, A: 'lt, H: GhostHandler<'lt, X, TestProtocol>> TestActorRef<'lt, X>
-    for GhostEndpointFull<'lt, TestProtocol, A, X, H>
+    for GhostEndpointFull<'lt, TestProtocol, A, X, H, GhostTestSystemRef<'lt>>
```

Comment:
This is supposed to represent auto-generated code... we don't want to hard-code a specific system here, so this should be bubbled up as a generic type here.